src/main/java/picard/analysis/CollectOxoGMetrics.java

```diff
@@ -313,7 +301,7 @@ protected int doWork() {
 
             // Skip sites at the end of chromosomes 
             final byte[] bases = refWalker.get(info.getSequenceIndex()).getBases();
-            if (pos < 3 || pos > bases.length - 3) continue;
+            if (pos <= CONTEXT_SIZE|| pos > bases.length - CONTEXT_SIZE) continue;
```

Comment:
yeah...I think it was wrong before! we only need to make sure that we are CONTEXT_SIZE bases away from the edges of the reference. a fixed value of 3 is incorrect in 2 ways: 

- it consistently avoids the second and second-to last bases in a reference (normally not a problem due to the N-masking in the ends)
- it would result in OOB exception if one runs it with CONTEXT_SIZE=5 (again, normally not a problem due to the lack of reads that manage to align so close to the edge of a contig)